docs/data/sql_functions.yml

```diff
@@ -580,9 +580,20 @@ json:
       This follows the ISO/IEC TR 19075-6 specification for JSON support in SQL.
 
       ```
-      SELECT JSON_EXISTS(
-        '{"items": [{"name": "Item 1"}]}',
-        'lax $.items[0].name' TRUE ON ERROR);
+      // TRUE
```

Comment:
I'm still not happy with the docs. As someone from outside, I still have a couple of questions. This doc should answer:

- Which path modes exists?
- What is the default path mode?
- What does every mode mean?
- The example implicitly shows that `FALSE ON ERROR` is the default strategy but we should mention this explicitly as well.

Reply:
I thought we discussed explicitly not to document the entirety of the specification but just refer to it?